src/main/java/org/opentripplanner/transit/raptor/api/transit/RaptorTransfer.java

```diff
@@ -47,4 +47,23 @@
      */
     int durationInSeconds();
 
+    /**
+     * Return the number publicly available services for this {@link RaptorTransfer}. This is used by Raptor to get the
```

Comment:
`services not part of the RAPTOR algorithm, such as flex or micromobility services, used as part of this RaptorTransfer `

Reply:
`Some services involving transfers are not handled by the RAPTOR algorithm and need to be inserted into the algorithm at a specific place of the algorithm. `

Reply:
`This method should only be overridden if an access or egress has services, such as flex, where the calculation happens outside the RAPTOR algorithm`

Reply:
Add a header comment above that these methods should be used only for special cases, such as flex or micromobility